python/kfserving/kfserving/test_storage.py

```diff
@@ -34,3 +37,31 @@ def test_no_prefix_local_path():
     relative_path = '.'
     assert kfserving.Storage.download(abs_path) == abs_path
     assert kfserving.Storage.download(relative_path) == relative_path
+
+
+def test_public_gcs():
+    gcs_path = 'gs://kfserving-samples/models/tensorflow/flowers'
+    assert kfserving.Storage.download(gcs_path)
+
+
+def test_private_gcs():
+    if os.getenv("GOOGLE_APPLICATION_CREDENTIALS", "") and os.getenv("GCS_PRIVATE_PATH", ""):
+        assert kfserving.Storage.download(os.getenv("GCS_PRIVATE_PATH", ""))
+    else:
```

Comment:
Same comment via mocking, our tests should not turn off if environment variables aren't set.